hangman.py

```diff
@@ -0,0 +1,256 @@
+# Problem Set 2, hangman.py
+# Name:Mizin Danilo
+# Collaborators:Piustonen
+# Time spent:
+
+import string
+import random
+
+
+WORDLIST_FILENAME = "words.txt"
+LIST_VOWELS = frozenset("aeiou")
+GAME_WITH_HINTS = "*"
+HIDDEN_LETTER = "_"
+GUESSES_POINT = 6
+WARNINGS_POINT = 3
+
+
+def load_words():
+    """
+    Returns a list of valid words. Words are strings of lowercase letters.
+
+    Depending on the size of the word list, this function may
+    take a while to finish.
+    """
+    print("Loading word list from file...")
+    # inFile: file
+    inFile = open(WORDLIST_FILENAME, 'r')
+    # line: string
+    line = inFile.readline()
+    # wordlist: list of strings
+    wordlist = line.split()
+    print("  ", len(wordlist), "words loaded.")
+    return wordlist
+
+
+def choose_word(wordlist):
+    """
+    wordlist (list): list of words (strings)
+
+    Returns a word from wordlist at random
+    """
+    return random.choice(wordlist)
+
+
+wordlist = load_words()
+
+
+def is_word_guessed(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing; assumes all letters are
+      lowercase
+    letters_guessed: list (of letters), which letters have been guessed so far;
+      assumes that all letters are lowercase
+    returns: boolean, True if all the letters of secret_word are in letters_guessed;
+      False otherwise
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string, comprised of letters, underscores (_), and spaces that represents
+      which letters in secret_word have been guessed so far.
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def get_available_letters(letters_guessed):
+    '''
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    '''
+    # FILL IN YOUR CODE HERE AND DELETE "pass"
+    pass
+
+
+def warnings(value_mistake, warnings_point, guesses_point, secret_word, letters_guessed):
+    """
+    Subtract warnings point and output name of error
+    """
+    if warnings_point > 0:
+        warnings_point -= 1
+        print(f"Oops! {value_mistake}", "\nYou have", warnings_point, "warnings left:",
```

Comment:
Could you have different prints? To avoid using special characters, which would be more readable